src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/PackageManagerUIRefreshEvent.cs

```diff
@@ -3,22 +3,25 @@
 
 using System;
 using NuGet.Common;
+using NuGet.VisualStudio.Internal.Contracts;
 
 namespace NuGet.PackageManagement.Telemetry
 {
     public sealed class PackageManagerUIRefreshEvent : TelemetryEvent
     {
         private const string EventName = "PMUIRefresh";
 
-        public PackageManagerUIRefreshEvent(
+        private PackageManagerUIRefreshEvent(
             Guid parentId,
             bool isSolutionLevel,
             RefreshOperationSource refreshSource,
             RefreshOperationStatus refreshStatus,
-            string tab,
+            ItemFilter tab,
```

Comment:
Are you sure the filter text is coming though, and not just the class name? (a class that doesn't override `ToString()` will just print out its own class name)

The type was changed, but not how the value is used.

Reply:
ItemFilter is an enum, so I think it prints the enum value. On my tests, it shows the name of the enum value.

![image](https://user-images.githubusercontent.com/1192347/207445842-86063a16-fe52-4827-ac94-36dac29b049d.png)

![image](https://user-images.githubusercontent.com/1192347/207448129-9fba2f3f-0989-4681-ad29-fe9ba7877857.png)

Reply:
So, yes, I'm confident the desired text is sent through telemetry.